### ISSUE TITLE
Client should have an option to be disabled

### ISSUE THREAD

**Issue by tabakerov** [matipan/metrics#1]
I believe it would be usefull to have a flag for Client constructor to disable actual data sending to DataDog (in case of testing)

**Comment by matias-pan-globant** [comment]
I agree. I'd think of adding a StopReport/StartReport that simply sets a boolean to true/false maybe?
The `Report` function can then check that to see if reporting is enabled or not. I'd leave `ReportLoop` as is, so that it continues to run and we only check for flag in the function that does the actual reporting

**Comment by tabakerov** [comment]
That could work but as we've discussed on Slack
- disabled reporting could produce memory leak (not an actual leak but...)
- client should handle empty configuration
